### ISSUE TITLE
Error with data(dis.data.re) and others

### ISSUE THREAD

**Issue by grwhumphries** [lindesaysh/MRSea#5]
Just found an error with this dataset:  
```
> data(dis.data.re)  # SURVEY DATA
> head(dis.data.re)
  transect.id transect.label season impact segment.id segment.label length  x.pos   y.pos   depth object distance
1           1              1      1      0          1           1-1  0.306 656.25 6043.75 -27.359     NA       NA
2           1              1      1      0          2           1-2  0.500 656.25 6044.25 -27.561     NA       NA
3           1              1      1      0          3           1-3  0.500 656.25 6044.75 -28.608     NA       NA
4           1              1      1      0          4           1-4  0.500 656.25 6045.25 -27.999     NA       NA
5           1              1      1      0          5           1-5  0.500 656.25 6045.75 -27.519     NA       NA
6           1              1      1      0          6           1-6  0.500 656.25 6046.25 -27.223     NA       NA
```

Basically, the x.pos and y.pos fields are not correct. I believe this is leading to errors in the tutorial

**Comment by lindesaysh** [comment]
Hi Grant,

Thanks for your message.  I am going to need a little more information before I can do anything with your query.  What version of MRSea are you using, which tutorial are you referring to and what are these errors that you speak of?  What is wrong with the x and y columns?

Kind Regards,
Lindesay

**Comment by grwhumphries** [comment]
Hi Lindesay, thanks for the reply :) 

I've got MRSea version 1.0.  The tutorial is one that you wrote in 2013:  https://tethys.pnnl.gov/sites/default/files/publications/Scott-Hayward-et-al-2013-b.pdf    

The tutorial and the version I'm using don't match up, which is what the problem is for the most part as far as I can tell.   In this issue I've just posted about  dis.data.re which has x.pos and y.pos values of 656.25 and 6043.75 (for example), These should be 6562500 and 60437500 (for example) - otherwise when you go to use the makeDists() function for the knot spline locations (using 2000) it implodes.
